### PR TITLE
FEE: Merge `browse-html+sass` into `master`

### PR DESCRIPTION
**Pull request by al-the-x**
TODO: Make a punchlist of what is being added...

### CODE REVIEW

**Comment by al-the-x** [comment]
FYI, check out commit e5b5075, which adds `bower_components/` and `npm-debug.log` to the repo. Looks like a case of a sloppy `git add .` if I had to guess.

File: angular/src/js/products.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    //         var self = this;
+    //         self.items = [];
+    //        
+    //    
+    //        
+    //       $http.get('js/etsy.json').succes(function(data) {
+    //           self.items = data;
+    //        
+    //    })
+    //    } ])
```
Comment (al-the-x) [inline angular/src/js/products.js]: Probably safe to delete this, if it's not being used.

File: angular/src/products.html [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+    <link rel="stylesheet" href="css/main.css">
+    <script src="js/vendor/modernizr-2.6.2.min.js"></script>
+    <!-- bower:js -->
+    <script src="../bower_components/jquery/dist/jquery.js"></script>
+    <script src="../bower_components/bootstrap/dist/js/bootstrap.js"></script>
+    <script src="../bower_components/angular/angular.js"></script>
+    <!-- endbower -->
+
+</head>
+<style type="text/css">
```
Comment (al-the-x) [inline angular/src/products.html]: Let's move this into `scss/products.scss`, which should produce a `css/products.css` file courtesy of `gulp sass`, and add a `<link>` tag to that generated `css/products.css` file.

File: angular/src/products.html [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+
+<body ng-app="BetterThanEtsy">
+
+    <div class="container">
+
+        <nav class="navbar">
+            <!--
+           <ul class="nav nav-pills col-xs-offset-8 col-sm-offset-10 col-md-offset-10" role="tablist">
+               <li role="presentation" class="active">
+-->
```
Comment (al-the-x) [inline angular/src/products.html]: Any of this commented code should be removed...

File: angular/src/products.html [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+                    <h1>{{item.title}}</h1>
+                    <span>{{item.price}}</span>
+                    <p>{{item.description}}</p>
+                    <button>Add to cart</button>
+                </div>
+
+            </div>
+            <!--this div is for the grid layout-->
+
+            <div class="tab-pane" id="list">
```
Comment (al-the-x) [inline angular/src/products.html]: Kinda hacky the way you're using the Bootstrap tabs here... What do you think about using `ng-show` and `ng-hide` on these elements instead? You'll need a property on `StoreController` to indicate whether the grid or list view is active.

File: bower_components/angular/.bower.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  "_resolution": {
+    "type": "version",
+    "tag": "v1.3.14",
+    "commit": "bdd91e1331b48ae89adfb8444998e290b774cc6d"
+  },
+  "_source": "git://github.com/angular/bower-angular.git",
+  "_target": "~1.3.14",
+  "_originalSource": "angular",
+  "_direct": true
+}
```
Comment (al-the-x) [inline bower_components/angular/.bower.json:18]: Not sure how you ended up with a `.bower.json` file _in addition_ to the `bower.json` file for the project...

**Comment by al-the-x** [comment]
In case it's not clear, I do want to see the corrections from my comments before I merge this PR into `master`... I think all the merge hockey we've done in class has effectively removed  `bower_components/`, but the others are still valid.

**Comment by al-the-x** [comment]
Please also add a checklist of working items and known issues to this PR. Thanks.

**Comment by GarrettGraham** [comment]
## Known Issues/ what to re-factore and change
- [x] hover states on the buttons and the glyphicons
- [x] color of the glyphicons
- [x] Sam: refactor bootstrap stuff to be more angular...
  - [x] use ng-show or ng-hide
  - [x] need a property on the StoreController to indicate whether the grid or list view is active
- [x] Carolina: refactor the tabs -- into Angular as a flag in the sand
- [x] run gulp to watch SCSS file in order for browser-sync
- [x] source links working in order to render code from the products.html, products.js and products.scss files vis-à-vis browser-sync
- [ ] remove superfluous links and code from products.html
  - [x] deleted commented out code if it is not beign used or will not be used
- [x] produce a css/products.css file by running gulp sass
  - [x] gulpfile functions correct for browser-sync
  - [x] `gulp server` to get browser-sync to render scss
- [x] .bower.json vs bower.json??
  - [x] delete the .bower.json file
- [x] toggle between grid view and list view
- [ ] populate data approptiately

**Comment by GarrettGraham** [comment]
## Working Items/Features
- [x] the gid layout is great!

File: angular/src/js/products.js (hunk 1 above)
Comment (carolinapowers) [inline angular/src/js/products.js]: It's working code...It show us how data looks ugly on our HTML  if nothing is done to render it properly.  But, yeah... It's being deleted and saved somewhere else. :)

File: angular/src/products.html (hunk 3 above)
Comment (carolinapowers) [inline angular/src/products.html]: Comments deleted. :)

File: angular/src/products.html (hunk 4 above)
Comment (carolinapowers) [inline angular/src/products.html]: We refactored it today and we are now using Angular. Tomorrow we will be working on refactoring Html and Css into Sass and hopefully we will be able to use Ng-Class to show the two different views.

File: angular/src/products.html (hunk 2 above)
Comment (carolinapowers) [inline angular/src/products.html]: We are not clear of what we should we move into the scss/products.scss...

**Pull request merged**